x-pack/plugins/monitoring/server/config.ts

```diff
@@ -51,7 +51,7 @@ export const configSchema = schema.object({
     }),
     min_interval_seconds: schema.number({ defaultValue: 10 }),
     show_license_expiration: schema.boolean({ defaultValue: true }),
-    render_react_app: schema.boolean({ defaultValue: false }),
+    render_react_app: schema.boolean({ defaultValue: true }),
```

Comment:
Probably better to put this in it's own PR off of master so we can use that to continually check how master is going on react-mode.